framework/src/controls/PIDTransientControl.C

```diff
@@ -147,7 +151,15 @@ PIDTransientControl::execute()
     if (_dt > 0)
       value += _Kder * delta / _dt;
 
-    value = std::min(std::max(_minimum_output_value, value), _maximum_output_value);
+    // If the maxmimum rate of change by the pid is fixed
+    if ((isParamSetByUser("maximum_change_rate")))
+    {
+      value = std::min(std::max(_minimum_output_value, value), _maximum_output_value);
+      value = std::min(std::max(_value_old - _dt * _maximum_change_rate, value),
```

Comment:
I dont think this is the right math for this
you still need to use the real change rate when it is underneath the max change rate in absolute value

Reply:
deleted line 157.

Reply:
the std::max is ensuring that it is above the lower limit and std::min is ensuring that it is below the upper limit. So it will be using the real change rate when it is under the max change rate. please let me know what you want me to change